cmd/allowance.go

```diff
@@ -16,12 +16,8 @@ var force bool
 var allowanceCmd = &cobra.Command{
 	Use:   "allowance",
 	Short: "Get your current Bonuslys for spending and giving away",
-	Long: `A longer description that spans multiple lines and likely contains examples
-and usage of using your command. For example:
-
-Cobra is a CLI library for Go that empowers applications.
-This application is a tool to generate the needed files
-to quickly create a Cobra application.`,
+	Long: `This command displays your bonus.ly account balances informing the respective ammounts
```

Comment:
Some typos here, please check again :)

Reply:
Typo fixed. I also reverted other minor change (bonus.ly -> Bonusly). I'll consider the original as default. My concern is how to distinguish between bonuslys (currency) and bonuslys (service).

Reply:
I like "bonus.ly" when referring to the service/website and "Bonusly" when referring to the currency.